src/components/atoms/TextInput.vue

```diff
@@ -29,26 +26,26 @@ export default {
   },
   components: {
     ChatOptionButton,
-  }, 
+  },
   emits: ["add-message"],
-  setup(_, context){
+  setup(_, context) {
     const text = "";
-    function sendText(){
-      if(this.text.length > 0){
-        context.emit("add-message", this.text)
+    function sendText() {
```

Comment:
Can this clear the text field as well?

Reply:
Also also, if we hit return can it click the send button?

Reply:
I can't get the text to clear but I added the keyup.enter event to send messages.